mailbox/api/src/main/java/org/apache/james/mailbox/indexer/ReIndexer.java

```diff
@@ -69,6 +69,7 @@ public static Builder builder() {
 
         public enum Mode {
             REBUILD_ALL("rebuildAll"),
+            REBUILD_ALL_NO_CLEANUP("rebuildAllNoCleanup"),
```

Comment:
Code change.

Mind blowing, but given the `if (...)` in ReindexerImpl it does what I want.

Reply:
You are actually right :)